msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -3,15 +3,17 @@
   <Fragment>
     <UI>
       <Dialog Id="GitChooseEnvironmentDlg" Width="370" Height="270" Title="!(loc.GitChooseEnvironmentDlg_Title)" X="50" Y="50">
-        <Control Type="RadioButtonGroup" Property="CLIENVIRONMENT" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
-          <RadioButtonGroup Property="CLIENVIRONMENT">
+        <Control Type="RadioButtonGroup" Property="UI_CliEnvironment" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
+          <RadioButtonGroup Property="UI_CliEnvironment">
```

Comment:
After seeing another of your comments below, I agree it would have been better to fix the original commit once I realized my misunderstanding/mistake.

However, introducing bits and pieces of the fully-functioning UI would've been harder. (At least, that's my opinion as the one having done the work. :wink: There aren't many graphical UI tools for WiX (at least, not many that are free); I did find one to use to help with the bulk of it, but that tool was very old and didn't have a great UX, and so there was much hand-editing of the generated UI XML.)